packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -33,39 +34,56 @@ class FlatTable<T> extends StatefulWidget {
   _FlatTableState<T> createState() => _FlatTableState<T>();
 }
 
-class _FlatTableState<T> extends NestingTableState<T, FlatTable<T>> {
-  @override
-  bool get startAtBottom => true;
+class _FlatTableState<T> extends State<FlatTable<T>> {
+  List<double> columnWidths = [];
 
   @override
-  TreeNode<TreeNode> asTreeNode(T node) => null;
+  void initState() {
+    super.initState();
+    columnWidths = computeColumnWidths();
+  }
 
   @override
-  List<ColumnData<T>> get columns => widget.columns;
+  void didUpdateWidget(FlatTable<T> oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    columnWidths = computeColumnWidths();
```

Comment:
maybe use the pattern of providing an 
_update or similar method that is called both by initState and didUpdateWidget.

Reply:
For now, this is the same.  If we need to update any other parts of the state when the widget is updated, that does become a useful refactor.